tests/turbo.node.test.ts

```diff
@@ -649,35 +650,95 @@ describe('Node environment', () => {
       });
     });
 
-    describe('upload()', () => {
-      const validDataItemOpts = [
-        {
-          target: '43charactersAbcdEfghIjklMnopQrstUvwxYz12345',
-          anchor: 'anchorMustBeThirtyTwoBytesLong!!',
-          tags: [
-            {
-              name: '', // empty name
-              value: '', // empty val
-            },
-          ],
+    const validDataItemOpts = [
+      {
```

Comment:
diff looks wild here; basically we share validDataItemOpts between both test suites now  and we skip the tag too long tests because theya re failing in some way vs latest upload images